### ISSUE TITLE
Importing dictionaries for two ligands with the same name in two different molecules break the bond representation

### ISSUE THREAD

**Issue by FilomenoSanchez** [moorhen-coot/Moorhen#3]

**Comment by LucreziaCatapano** [comment]
we could not reproduce this problem. Maybe is fixed! If not, please provide us more details (data) on how to reproduce.

**Comment by FilomenoSanchez** [comment]
I wrote the test at line 908 of moorhenMolecule.test.js to test this [here](https://github.com/moorhen-coot/Moorhen/blob/f8113aac859363327857646fe4a3e0596920626b/baby-gru/tests/__tests__/moorhenMolecule.test.js#L908). It is currently failing, it should give you a way to reproduce this error using `molecules_container`. Alternatively, using the UI the fastest way to get "new ligands" with the same name is using the "From SMILES" utility:
1. Load tutorial data 1 and pan the view somewhere away of the molecule for good visibility
2. Ligand -> From SMILES... 
3. Create a ligand named "LIG" using smiles "C1CCCCC1" and choose "mol-1" in the selector under "Make monomer available to". Leave the rest with default values and click OK.
4. Create a second ligand also named "LIG" using smiles "O=N(=O)C1=CC=CC=C1" and choose "mol-1" in the selector under "Make monomer available to". Leave the rest with default values and click OK.
5. See the results attached in screenshot. Left is the current output of the program, right is the expected output. The only difference between these two screenshots was that on the left I named bot ligands "LIG" and on the right I named them "LIG" and "XXX".
![Screenshot from 2024-01-17 18-35-07](https://github.com/moorhen-coot/Moorhen/assets/17200301/aa797ca2-e8c6-41b1-a914-8528859400e3)
